CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # GoodData Ruby SDK Changelog
+## 2.1.15
+ - FEATURE: BCO-1119 Introduce data sources API
+
```

Comment:
We should mention Rails 6.1 support:
Upgrade activesupport to be able used with Rails 6.1

Reply:
Done update change logs